modules/introduction/pages/whats-new.adoc

```diff
@@ -64,3 +64,43 @@ These roles are listed at xref:learn:security/roles.adoc[Roles].
 
 TLS 1.3 cipher-suites can now by used by all services; and by the Cluster Manager, XDCR, and Views.
 For information, see xref:learn:security/on-the-wire-security.adoc#tls-3-cipher-suite-limitations[On the Wire Security].
+
+=== ARM Support
+
+Amazon Linux is now supported on the ARM platform.
+See xref:release-notes:relnotes.adoc[] and xref:install:install-platforms.adoc[]
+
+=== Magma Storage Engine
+[.edition]#{enterprise}#
+
+The new Magma Storage Engine has been added to 7.1 as an Enterprise Edition feature, allowing for higher performance with very large datasets. 
+Magma is a disk-based engine, so is highly suited to datasets that will not fit in available memory.
+You can find more details on Magma in xref:learn:buckets-memory-and-storage/storage-engines.adoc[]
+
+=== Improved Failover
+
+Automatic Failover can now failover several nodes concurrently and automatic failover of index services is now supported. 
+Improvements have also been made to the rebalancing algorithms so that odd numbers of vbuckets will be better distributed across server groups. 
```

Comment:
This sentence is incorrect.
Replace with:
Improvements have also been made to rebalancing algorithms so that active and all replicas will be spread across different server groups even when server groups are unequal.